examples/notebooks/plot_binary_classification_COMPAS.py

```diff
@@ -18,6 +16,8 @@
 #
 # To start, let's download the dataset using `tempeh`
 
+print(__doc__)
```

Comment:
Why is this being moved?

Reply:
Because there's no evident need to have it at the top AFAIK.